api/python/quilt3/data_transfer.py

```diff
@@ -429,19 +431,25 @@ def __init__(self, s3_client_provider, progress, done, run):
         self.run = run
 
 
-def _copy_file_list_internal(file_list, message, callback):
+@retry(stop=stop_after_attempt(MAX_COPY_FILE_LIST_RETRIES),
+       wait=wait_exponential(multiplier=1, min=1, max=10),
+       reraise=True)
+def _copy_file_list_internal(file_list, results, message, callback):
     """
```

Comment:
Please add a note in this comment to explain the results parameter. I think it's a container that's passed in to collect results?